components/homme/src/share/control_mod.F90

```diff
@@ -253,6 +253,21 @@ module control_mod
   real (kind=real_kind), public :: dcmip16_mu_q    = -1       ! additional uniform viscosity (scalar tracers); -1 implies it defaults to dcmip16_mu_s value
   real (kind=real_kind), public :: interp_lon0     = 0.0d0
 
+!PLANAR
+  real (kind=real_kind), public :: Lx, Ly !domain size
+  real (kind=real_kind), public :: Sx, Sy !shift
+  real (kind=real_kind), private, parameter :: tol_zero=1e-10 !tolerance to determine if lx,ly,sx,sy are set
```

Comment:
Should these live in control_mod or in physical_constants (where they were before)? I put them in physical constants since that is where rearth and rrearth live, and Lx/Ly are like that for planar. I'm okay with either choice though.